src/postgres/getCatalog.js

```diff
@@ -75,7 +75,22 @@ const addTables = (client, catalog) =>
     select
       n.nspname as "schemaName",
       c.relname as "name",
-      d.description as "description"
+      d.description as "description",
+      case when
+           (pg_catalog.pg_relation_is_updatable(c.oid, true)::bit(8) & B'00010000') = B'00010000'
```

Comment:
Oh wow, this looks like a nice function. Could I see its documentation? Also if you could include a link to the documentation in the code that would be amazing :blush: